pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/service/onchistory/CodeStudyColumnsProvider.java

```diff
@@ -0,0 +1,76 @@
+package org.broadinstitute.dsm.service.onchistory;
+
+import static org.broadinstitute.dsm.statics.DBConstants.ACCESSION_NUMBER;
+import static org.broadinstitute.dsm.statics.DBConstants.DATE_PX;
+import static org.broadinstitute.dsm.statics.DBConstants.DDP_ONC_HISTORY_DETAIL_ALIAS;
+import static org.broadinstitute.dsm.statics.DBConstants.DESTRUCTION_POLICY;
+import static org.broadinstitute.dsm.statics.DBConstants.FACILITY;
+import static org.broadinstitute.dsm.statics.DBConstants.FAX;
+import static org.broadinstitute.dsm.statics.DBConstants.FIELD_SETTINGS_ALIAS;
+import static org.broadinstitute.dsm.statics.DBConstants.HISTOLOGY;
+import static org.broadinstitute.dsm.statics.DBConstants.LOCATION_PX;
+import static org.broadinstitute.dsm.statics.DBConstants.PHONE;
+import static org.broadinstitute.dsm.statics.DBConstants.REQUEST;
+import static org.broadinstitute.dsm.statics.DBConstants.TYPE_PX;
+
+import java.util.LinkedHashMap;
+import java.util.Map;
+
+import org.broadinstitute.dsm.exception.DSMBadRequestException;
+
+// TODO temporary for development, will store in DB or config
+public class CodeStudyColumnsProvider implements StudyColumnsProvider {
+    public Map<String, OncHistoryUploadColumn> getColumnsForStudy(String realm) {
+        if (realm.equals("osteo2")) {
+            return getOsteoColumns();
```

Comment:
Naming nit: consider using getOsteo2Columns to disambiguate between "Osteo" and "Osteo2" since our code has other references to Osteo with qualifiers for different cases.

Reply:
Will do.